src/store/notes/index.ts

```diff
@@ -44,3 +53,30 @@ export const handleGetNotes =
 
     dispatch(saveNotes(normalizedNotes));
   };
+
+export const handleCreateNote =
+  (title: string, content: RawDraftContentState) =>
+  async (dispatch: AppDispatch): Promise<void> => {
+    const response = await rest.post('http://localhost:3000/api/notes', { title, content });
+
+    const normalizedNotes: TNote = {
+      id: response._id,
+      title: response.title,
+      content: convertToRaw(ContentState.createFromText(''))
+    };
+
+    dispatch(addNote(normalizedNotes));
+  };
+
+export const handleDeleteNote =
+  (id: string) =>
+  async (dispatch: AppDispatch): Promise<void> => {
+    const response = await rest.delete(`http://localhost:3000/api/notes/${id}`);
+    dispatch(deleteNote(id));
+  };
+
+export const handleUpdateNote =
+  (id: string | null, content: RawDraftContentState, title?: string) =>
+  async (dispatch: AppDispatch): Promise<void> => {
+    const response = await rest.put(`http://localhost:3000/api/notes/${id}`, { title, content });
```

Comment:
you should update exists note and get the new array of notes from database.
You have for this `handleGetNotes`